android/src/main/java/com/iterable/reactnative/RNIterableAPIModule.java

```diff
@@ -88,19 +88,19 @@ public void setEmail(@Nullable String email) {
 
     @ReactMethod
     public void setEmail(@Nullable String email, @Nullable String token) {
-        IterableLogger.d(TAG, "setEmail: " + email + " token: " + token);
+        IterableLogger.d(TAG, "setEmail: " + email + ", token: " + token);
         IterableApi.getInstance().setEmail(email, token);
     }
 
     @ReactMethod
     public void updateEmail(String email) {
-        IterableLogger.d(TAG, "Update Email: " + email);
+        IterableLogger.d(TAG, "updateEmail: " + email);
         IterableApi.getInstance().updateEmail(email);
     }
 
     @ReactMethod
     public void updateEmail(String email, @Nullable String token) {
-        IterableLogger.d(TAG, "updateEmail: " + email + " token: " + token);
+        IterableLogger.d(TAG, "updateEmail: " + email + ", token: " + token);
```

Comment:
Woah! Totally missed the commas and question marks in PR. Could have broken the code 😅